LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/LoRaModuleClient.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer.BasicsStation.ModuleConnection
+{
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Shared;
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public sealed class LoRaModuleClient : ILoraModuleClient
+    {
+        private ModuleClient moduleClient;
+
+        public uint OperationTimeoutInMilliseconds { get => this.moduleClient.OperationTimeoutInMilliseconds; set => this.moduleClient.OperationTimeoutInMilliseconds = value; }
```

Comment:
Remove as this appears to be unused:

```suggestion
```

Reply:
There is a reference to this on ModuleConnectionHost L73. As it's an interface and we needed this variable.. (it was not needed before as this was a uint)

```
if (networkServerConfiguration.IoTEdgeTimeout > 0)
            {
                this.loRaModuleClient.OperationTimeout = TimeSpan.FromMilliseconds(networkServerConfiguration.IoTEdgeTimeout);
                Logger.Log($"Changing timeout to {networkServerConfiguration.IoTEdgeTimeout} ms", LogLevel.Debug);
            }
```